specification/metadata/metadata_schema_column_definition.md

```diff
@@ -0,0 +1,87 @@
+# FOCUS Metadata Schema Column Definition
+The FOCUS metadata schema column definition provides a list of the columns present in the FOCUS dataset along with metadata about the columsn. The intent of the schema metadata is to aid pracitioners to understand the structure of the FOCUS data, without the need to parse the included dataset. 
+
+
+## Logical Section: Schema 
+### Subsection: Schema Column Definition
+
+## Schema Definition Metadata
+
+ * ### column_name
```

Comment:
Should this be an ID to align to the spec?

Reply:
Fixed.